addons/sale/i18n/sale.pot

```diff
@@ -830,8 +830,7 @@ msgid ""
 "Quotation: the document will be sent to and accessible by customers at any time.\n"
 "e.g. this option can be useful to share Product description files.\n"
 "Confirmed order: the document will be sent to and accessible by customers.\n"
-"e.g. this option can be useful to share User Manual or digital content bought on ecommerce. \n"
-"Inside quote: The document will be included in the pdf of the quotation between the header pages and the quote table. "
+"e.g. this option can be useful to share User Manual or digital content bought on ecommerce. "
```

Comment:
The exports were probably done on an instance with all modules installed.
(this one here comes from the module sale_pdf_quote_builder).

Remove the diff unrelated to your changes to be sure we don't impact anything else (only keep the unrelated python-format removal below, was not removed in a fw-port when it should have been).